terraform/modules/grafana/variables.tf

```diff
@@ -21,3 +21,7 @@ variable "vpc_cidr" {
 variable "ecr_url" {
   type = string
 }
+
+variable "sns_topic_arn" {
```

Comment:
Can we be more specific with this name? What's its' purpose as far as the Grafana setup is concerned?